php/auth.php

```diff
@@ -0,0 +1,49 @@
+<?php
+$ERRORLOG = getenv('PHP_ERROR_LOG');
```

Comment:
That `ini_get` doesn't work either. I propose setting `$ERRORLOG` equal to `"/var/log/nginx/pihole_error.log"` or something that would not rely on Pi-hole Core.